src/parser/scanner.lex

```diff
@@ -17,6 +17,8 @@ using TokenType = nebula::GraphParser::token;
 
 static constexpr size_t MAX_STRING = 4096;
 
+static constexpr size_t MAX_INTEGER = 9223372036854775808ULL;
```

Comment:
change a name

Reply:
What name?

Reply:
ABS_MIN_INTEGER?

Reply:
MIN not MAX?

Reply:
MIN, `ABS` is to describe the `MIN_INTEGER`.

Reply:
I don't it's clear, I think maybe `MAX_ABS_INTEGER` better.

Reply:
Good, I agree.